src/pipeline-server/docker-compose.pipeline-server.yml

```diff
@@ -40,10 +40,44 @@ services:
       - ${RETAIL_USE_CASE_ROOT:-..}/performance-tools/sample-media:/home/pipeline-server/sample-media
     networks:
       checkout-network: {}
+  camera-simulator1:
+    image: jrottenberg/ffmpeg:4.1-alpine
+    container_name: camera-simulator1
+    entrypoint: ["/bin/sh","-c"]
+    command: 
+    - | 
+       if [ ! -f /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 ]; then
+          mkdir -p /home/pipeline-server/sample-media
+          wget -O /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 https://www.pexels.com/download/video/4465029
+       fi
+       ffmpeg -nostdin -re -stream_loop -1 -i /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 -c copy -f rtsp -rtsp_transport tcp rtsp://camera-simulator:8554/camera_1
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ${RETAIL_USE_CASE_ROOT:-..}/performance-tools/sample-media:/home/pipeline-server/sample-media
+    networks:
+      checkout-network: {}
+  camera-simulator2:
+    image: jrottenberg/ffmpeg:4.1-alpine
+    container_name: camera-simulator2
+    entrypoint: ["/bin/sh","-c"]
+    command: 
+    - | 
+       if [ ! -f /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 ]; then
+          mkdir -p /home/pipeline-server/sample-media
+          wget -O /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 https://www.pexels.com/download/video/4465029
+       fi
+       ffmpeg -nostdin -re -stream_loop -1 -i /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 -c copy -f rtsp -rtsp_transport tcp rtsp://camera-simulator:8554/camera_2
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ${RETAIL_USE_CASE_ROOT:-..}/performance-tools/sample-media:/home/pipeline-server/sample-media
+    networks:
+      checkout-network: {}
 
   evamclient0:
     container_name: evam_0
-    image: amr-registry.caas.intel.com/nex-microservices/intel/edge_video_analytics_microservice@sha256:5588f0488d908bc07daab9555c2776eb4a9bead78f43dde2eec452b610a534b4
+    image: bmcginn/cv-workshop:2.1
```

Comment:
when are we building and naming this docker image?

Reply:
Good catch, I need to revert the images